src/dashboards/about.json

```diff
@@ -0,0 +1,192 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": "-- Grafana --",
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 0,
+    "id": null,
+    "links": [],
+    "panels": [
+      {
+        "datasource": null,
+        "fieldConfig": {
+          "defaults": {},
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 11,
+          "w": 18,
+          "x": 0,
+          "y": 0
+        },
+        "id": 2,
+        "options": {
+          "content": "# About Helm\n\nHELM is an application that allows users to create flexible dashboards to interact with data stored by OpenNMS. HELM works with Grafana to display and customize data from OpenNMS. Create and customize dashboards to display the data you want to see - alarms, outages, key performance indicators - in a way that best  meets the needs of your workflow and staff.\n\n## Datasources\n\n * Performance - retrieve performance metrics (i.e., data collection and/or response time metrics) from the Measurements API in OpenNMS.\n * Flow - retrieve application- and conversation-related metrics derived from flow data stored by OpenNMS.\n * Entity - retrieve entity model objects from OpenNMS. It currently supports alarms and nodes.\n\n## Example Dashboards\n\n * https://grafana.com/grafana/dashboards?search=opennms\n * https://github.com/opennms-forge/grafana-dashboards\n ",
```

Comment:
A nitpick, but there's really nowhere else (other than Jira issue names) that we refer to helm in all-caps.  I'd change it to always use "Helm"

Reply:
I actually copied that verbatim from the documentation.

Reply:
Oh weird. I guess I am wrong then. Guess we need to figure out what the branding is suppose to be then 'cause I've never referred to it that way publicly in anything. 😅